tests/v2/validation/projects/projects_additional_test.go

```diff
@@ -0,0 +1,131 @@
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ProjectsAdditionalTestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+	cluster *management.Cluster
+}
+
+func (pr *ProjectsAdditionalTestSuite) TearDownSuite() {
+	pr.session.Cleanup()
+}
+
+func (pr *ProjectsAdditionalTestSuite) SetupSuite() {
+	pr.session = session.NewSession()
+
+	client, err := rancher.NewClient("", pr.session)
+	require.NoError(pr.T(), err)
+
+	pr.client = client
+
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(pr.T(), clusterName, "Cluster name to install should be set")
+	clusterID, err := clusters.GetClusterIDByName(pr.client, clusterName)
+	require.NoError(pr.T(), err, "Error getting cluster ID")
+	pr.cluster, err = pr.client.Management.Cluster.ByID(clusterID)
+	require.NoError(pr.T(), err)
+
+}
+
+func (pr *ProjectsAdditionalTestSuite) TestUserAdditionToClusterWithTerminatingProjectNamespace() {
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	testStartTime := time.Now()
+
+	log.Info("Create a standard user.")
+	createdUser, err := users.CreateUserWithRole(pr.client, users.UserConfig(), standardUser)
+	require.NoError(pr.T(), err)
+	pr.T().Logf("Created user: %v", createdUser.Username)
+
+	log.Info("Create a project in the downstream cluster.")
+	createdProject, err := createProject(pr.client, pr.cluster.ID)
+	require.NoError(pr.T(), err)
+	err = waitForFinalizerToUpdate(pr.client, createdProject.Name)
+	require.NoError(pr.T(), err)
+
+	log.Info("Simulate a project stuck in terminating state by adding a finalizer to the project.")
+	finalizer := append([]string{dummyFinalizer}, createdProject.Finalizers...)
+	_, err = updateProjectNamespaceFinalizer(pr.client, createdProject, finalizer)
+	require.NoError(pr.T(), err, "Failed to update finalizer.")
+
+	log.Info("Delete the Project.")
+	err = projectsApi.DeleteProject(pr.client, createdProject)
+	require.NoError(pr.T(), err)
+
+	log.Info("Add the standard user to the downstream cluster as cluster owner.")
+	err = users.AddClusterRoleToUser(pr.client, pr.cluster, createdUser, roleOwner, nil)
+	require.NoError(pr.T(), err)
+
+	log.Info("Verify that there are no errors in the Rancher logs related to role binding.")
+	leaderPodName, err := rancherleader.GetRancherLeaderPod(pr.client)
+	require.NoError(pr.T(), err)
+	errorRegex := `\[ERROR\] error syncing '(.*?)': handler mgmt-auth-crtb-controller: .*? (?:not found|is forbidden), requeuing`
+	err = checkPodLogsForErrors(pr.client, localCluster, leaderPodName, RancherNamespace, errorRegex, testStartTime)
+	require.NoError(pr.T(), err)
+
+	log.Info("Remove the finalizer that was previously added to the project.")
+	finalizer = nil
+	_, err = updateProjectNamespaceFinalizer(pr.client, createdProject, finalizer)
+	require.NoError(pr.T(), err, "Failed to update finalizer.")
```

Comment:
Should we check for any errors in the rancher pod for a crtb after the project is cleaned up?